src/NzbDrone.Core.Test/NotificationTests/EmailTests/EmailSettingsValidatorFixture.cs

```diff
@@ -62,7 +62,6 @@ public void should_not_be_valid_if_from_is_empty()
         }
 
         [TestCase("sonarr")]
-        [TestCase("sonarr@sonarr")]
```

Comment:
FluentValidation 9 switched to .NET Core’s `EmailAddressAttribute`:
https://docs.fluentvalidation.net/en/latest/upgrading-to-9.html

`"sonarr@sonarr"` is actually a valid email address:
https://en.wikipedia.org/wiki/Email_address#Examples